src/components/button/theme.css

```diff
@@ -502,7 +502,7 @@
 
 .spinner {
   left: 50%;
-  position: absolute;
+  position: absolute !important;
```

Comment:
Is this also an overwrite for another repository?

Reply:
Looks like it 😕

Reply:
I propose we merge this for now as it is actually broken in core, but we add a `@TODO` comment to atleast know it needs to be fixed. I looked into it for a bit but I couldn't find the cause.

Reply:
Thanks for the feedback! Let's get this merged for now!